src/test/resources/application.yml

```diff
@@ -21,3 +21,11 @@ management:
 logging:
   level:
     org.springframework.web: debug
+application:
+  minio-remote:
```

Comment:
the same https://github.com/folio-org/mod-bulk-operations/pull/9/files#r1041913599